test/services/dumper/dumper-sequelize.test.js

```diff
@@ -5,6 +5,7 @@ const simpleModel = require('../../../test-expected/sequelize/db-analysis-output
 const belongsToModel = require('../../../test-expected/sequelize/db-analysis-output/addresses.expected.json');
 const otherAssociationsModel = require('../../../test-expected/sequelize/db-analysis-output/users.expected.json');
 const exportModel = require('../../../test-expected/sequelize/db-analysis-output/export.expected.json');
+const defaultValuesModel = require('../../../test-expected/sequelize/db-analysis-output/default-values.expected.js');
```

Comment:
This looks like a fixture more than an expected value. However, I see that this whole file use the same pattern, so I guess it's OK.

Reply:
I had to use a JS file to because some expressions where needed (eg: `"defaultValue": Sequelize.literal('now()')`). Could not make this work with a fixture / JSON file.
Definitely used a fixture as the code base for this.